website/content/en/docs/building-operators/helm/tutorial.md

```diff
@@ -1,26 +1,27 @@
 ---
 title: Tutorial for Helm-based Operators
 linkTitle: Tutorial
-weight: 200
-description: A simple set of instructions that demonstrates the basics of setting up and running a Helm-based operator.
+weight: 2
```

Comment:
```suggestion
weight: 200
```

Reply:
I changed to show the reference at the end.